apps/TutorialServo/TutorialServo.cpp

```diff
@@ -0,0 +1,38 @@
+#include "TutorialServo.h"
+#include "mbed.h"
+
+float minPulsewidthInMs, maxPulsewidthInMs;
```

Comment:
an important idea in object oriented programming is encapsulation. its not a good idea to declare these vars globally. you should think about how to limit access of these vars only to the class.